src/plugins/cosmos/components/modals/UnstakingConfirm/views/Confirm.tsx

```diff
@@ -0,0 +1,159 @@
+import { InfoOutlineIcon } from '@chakra-ui/icons'
+import { Flex } from '@chakra-ui/layout'
+import { Button, FormControl, ModalHeader, Text as CText, Tooltip } from '@chakra-ui/react'
+import { CAIP19 } from '@shapeshiftoss/caip'
+import { chainAdapters } from '@shapeshiftoss/types'
+import { Asset } from '@shapeshiftoss/types'
+import { AnimatePresence } from 'framer-motion'
+import { TxFeeRadioGroup } from 'plugins/cosmos/components/TxFeeRadioGroup/TxFeeRadioGroup'
+import { FormProvider, useForm } from 'react-hook-form'
+import { useTranslate } from 'react-polyglot'
+import { useHistory } from 'react-router-dom'
+import { Amount } from 'components/Amount/Amount'
+import { Text } from 'components/Text'
+import { useModal } from 'context/ModalProvider/ModalProvider'
+import { BigNumber } from 'lib/bignumber/bignumber'
+
+import { UnstakingPath } from '../UnstakingConfirm'
+
+export enum InputType {
+  Crypto = 'crypto',
+  Fiat = 'fiat'
+}
+
+export enum Field {
+  FeeType = 'feeType'
+}
+
+export type UnstakingValues = {
+  [Field.FeeType]: chainAdapters.FeeDataKey
+}
+
+type UnstakeProps = {
+  assetId: CAIP19
+  fiatRate: BigNumber
+  cryptoUnstakeAmount: BigNumber
+}
+
+const DEFAULT_VALIDATOR_NAME = 'Shapeshift Validator'
+
+// TODO: Wire up the whole component with staked data
+export const Confirm = ({ assetId, cryptoUnstakeAmount, fiatRate }: UnstakeProps) => {
+  const methods = useForm<UnstakingValues>({
+    mode: 'onChange',
+    defaultValues: {
+      [Field.FeeType]: chainAdapters.FeeDataKey.Average
+    }
+  })
+
+  const { handleSubmit } = methods
+
+  const { cosmosUnstakingConfirm } = useModal()
+
+  const memoryHistory = useHistory()
+  const onSubmit = (_: any) => {
+    memoryHistory.push(UnstakingPath.Broadcast)
+  }
+
+  const translate = useTranslate()
+
+  const handleCancel = () => {
+    cosmosUnstakingConfirm.close()
+  }
+
+  // TODO: wire me up, parentheses are nice but let's get asset name from selectAssetNameById instead of this
+  const asset = (_ => ({
+    name: 'Osmosis',
+    symbol: 'OSMO',
+    caip19: assetId,
+    chain: 'osmosis'
+  }))(assetId) as Asset
+  return (
+    <FormProvider {...methods}>
+      <AnimatePresence exitBeforeEnter initial={false}>
+        <Flex
+          as='form'
+          pt='14px'
+          pb='18px'
+          px='30px'
+          onSubmit={handleSubmit(onSubmit)}
+          direction='column'
+          alignItems='center'
+          justifyContent='space-between'
```

Comment:
```suggestion
          justify='space-between'
```

Reply:
There are others where shorthand are not used, feel free to change them too

Also: 

It needs to be defined, do we use shorthand everywhere, or do we use real properties everywhere?

Reply:
I prefer we use the full name of the css property, justify/justifyContent, align/alignItems, etc...

I think this helps make it super clear that this is a flexbox element, vs a table element or something along those lines that also takes align props.

Reply:
Yeah alright, but we should be consistent with this choice then :)